tensorflow/lite/delegates/gpu/delegate.cc

```diff
@@ -95,15 +101,7 @@ class Delegate {
   int num_delegate_kernels() const { return num_delegate_kernels_; }
 
  private:
-  TfLiteDelegate delegate_ = {
-      .data_ = reinterpret_cast<void*>(this),
-      .Prepare = DelegatePrepare,
-      .CopyFromBufferHandle = nullptr,
-      .CopyToBufferHandle = nullptr,
-      .FreeBufferHandle = nullptr,
-      .flags = kTfLiteDelegateFlagsNone,
-  };
-
+  TfLiteDelegate delegate_;
```

Comment:
doesn't MSVC support these syntax? What's the error? Maybe you need to enable C++11 or C++14?

Reply:
No. MSVC 2019 requires c++20 for this syntax. (I think it is because this syntax from c++20). But not all dependencies ready yet (like Eigen for example)